agents/locobot/locobot_agent.py

```diff
@@ -123,6 +125,15 @@ def objects_in_memory(sid):
             self.dashboard_memory["objects"] = objects
             sio.emit("updateState", {"memory": self.dashboard_memory})
 
+        @sio.on("store session id")
+        def create_loggings_object(sid, id):
+            filePath = "interaction_loggings_" + id + ".json"
+            self.interactionLogger = InteractionLogger(filePath)
```

Comment:
nit (and this applies to this whole file): snake case for python

see https://www.python.org/dev/peps/pep-0008/#function-and-variable-names